pallets/crowdloans/src/lib.rs

```diff
@@ -504,10 +489,7 @@ pub mod pallet {
                 amount,
                 false,
             )?;
-            TotalReserves::<T>::try_mutate(|b| -> DispatchResult {
-                *b = b.checked_add(amount).ok_or(ArithmeticError::Overflow)?;
-                Ok(())
-            })?;
+            T::XCM::update_total_reserves(amount)?;
```

Comment:
1/3
`update_total_reserves` invoked in crowdloan pallet, and also the transferring happens here

Reply:
@MrPai Are you saying that Asset transfer should also happen inside `update_total_reserves`?

Reply:
yes, that's what I mean